src/main/java/org/apache/sysml/runtime/instructions/cp/ListObject.java

```diff
@@ -44,30 +46,44 @@ public ListObject(List<Data> data, List<String> names) {
 		_data = data;
 		_names = names;
 	}
-	
+
+	public boolean isParameterizedList() {
```

Comment:
Please rename to NamedList - a list object can be either an unnamed or a named list.